### DISCUSSION TITLE
Stack function in violin plot

### DISCUSSION THREAD

**Discussion by Chanukya-Pavan** [satijalab/seurat#7919] [General]
Hi I was trying to get a graph with multiple genes in one violin plot figure, for which I used the stack function as shown below in the image and received the error as shown.
![image](https://github.com/satijalab/seurat/assets/112158036/6ea374de-dfc9-49fc-8a74-4d9d201762fe)
Can I get any suggestions on resolving this error

Thanks,
Chanukya

**Comment by shuai-zhong-pig** [comment]
i think add "ncol = 2" may help you, cause you add stack = T but not tell number of polt in one line. and i suggest do not add too much of arguments once, may first add no arguments make the plot is ok in base
